Sources/CLI/RunCommand.swift

```diff
@@ -294,6 +302,39 @@ struct ProcessIO {
         )
     }
 
+    static func streamStdin(
+        from: OSFile,
+        to: OSFile,
+        buffer: UnsafeMutableBufferPointer<UInt8>,
+        onErrorOrEOF: () -> Void,
+    ) {
+        while true {
+            let r = from.read(buffer)
```

Comment:
destructure into `(bytesRead, action)` for readability

or at least rename `r.read` to `r.bytesRead`, `r.readCount`, etc